.github/workflows/pr-check.yml

```diff
@@ -11,12 +11,24 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v4
-    - name: Make Tests env
+    - name: Make env
       run: |
         python3 -m venv .env
-        source ./.env/bin/activate
+        source .env/bin/activate
         pip install -r requirements.txt
     - name: Run Tests
       run: |
-        source ./.env/bin/activate
+        source .env/bin/activate
         pytest -vv --color=yes
+    - name: Black - Code Formatting
+      run: |
+        source .env/bin/activate
+        black \
+          --check \
+          --extend-exclude "(task1|task2|task3|task4|task5)" \
+          .
+      continue-on-error: false
```

Comment:
The `continue-on-error` setting is set to `false` be default and can be omitted here.

Why? in some steps you set this setting to `false`, but in other ones -- not.  Let's use the consistent approach (and add this setting in each step or remove it completely)